css/main.css

```diff
@@ -293,7 +441,7 @@ nav {
 
 .sponsors .cropped {
   position: relative;
-  width: 1440px;
+  width: 100%;
```

Comment:
I actually hardcoded 1440px for a reason so that the colored lines match with the image.  In order to make this work for screens > 1440px, please use max-width: 1440px instead.